### PR TITLE
Basic Page Set Up

### PR DESCRIPTION
**Pull request by DanSam5K**
## In this project milestone I completed the following:

- Set-up general requirements with .gitflow/workflows linter file
- Add linter files for HTML, CSS & JavaScript
- Set-up project  to use webpack
- Add HTML layout page
- Dynamically Inject hard-coded content and load it to the page with Javascript.
- Add basic CSS style to the loaded content to achieve basic page layout
- Fix all linter errors and minor changes
- Add descriptive documentation with screenshots

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,76 +1,71 @@
...
-### Deployment
-
-
-
-## Authors
+To get a local copy up and running follow these simple example steps.
+- Open terminal
+- Clone this project by the command `git clone https://github.com/DanSam5K/LeaderBoard-APIS.git`
+- `cd <clone>` folder
+- Run `npm start` in your local browser or using Live Server in Visual Studio Code.
```
Comment (voscarmv) [inline README.md]: - [ ] Please include the `npm install` instruction here.

**Review by voscarmv — CHANGES_REQUESTED** [review]
Hi @DanSam5K , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️

- Please read my comments on your README file and fix all issues.
- Please make sure your project includes a `dist` folder containing an `index.html` and `index.js` or `main.js`. That is: build the project and push the build folder to your repo.

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:

- Fixing the broken Live Demo link on your README

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 1 above)
Comment (DanSam5K) [reply README.md]: - Noted!

Comment (rloterh) [reply README.md]: This issue is well resolved! ✔️

File: .vscode/settings.json [hunk 2]
```diff
@@ -0,0 +1,8 @@
+{
```
Comment (rloterh) [inline .vscode/settings.json]: - [x] OPTIONAL
Please endeavor to keep this file's directory in the .gitignore file so as to keep your project structure clean

**Review by rloterh — APPROVED** [review]
## Status: Approved :heavy_check_mark:

Hi Daniel, 

Your project is complete! Great job for making the necessary changes recommended by the previous reviewer. There is nothing else to say other than... it's time to merge it 🍾 :shipit: .
Congratulations! 🎉🥂

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

_Kindly consider my comment(s) marked [OPTIONAL] under this review:_

<br>
Cheers and Happy coding!👏 💻 🍷 

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have 1 more limited review per this project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 1 above)
Comment (DanSam5K) [reply README.md]: @rloterh Thank you 🤝

File: .vscode/settings.json (hunk 2 above)
Comment (DanSam5K) [reply .vscode/settings.json]: - Noted I will not forget that thank you! 🤝

**Pull request merged**